.github/workflows/connector-performance-command.yml

```diff
@@ -4,7 +4,13 @@ on:
     inputs:
       connector:
         description: "Airbyte Connector"
+        type: choice
```

Comment:
Nice!
Can this replace the code down the file that filters for supported connectors - the big ugly if

Reply:
I think we still need that because we still want to trigger it from comment